src/main/kotlin/io/spotfire/solver/Application.kt

```diff
@@ -0,0 +1,16 @@
+package io.spotfire.solver
+
+import io.spotfire.solver.plugins.*
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+
+fun main() {
```

Comment:
We might be able to use [Hocon](https://ktor.io/docs/configuration-file.html#environment-variables) configuration here:

```hocon
// application.conf
ktor {
  application {
    port = ${?PORT} 8086
    host = ${?HOST} "0.0.0.0"
  }
}
```

```kotlin
fun main() {
    val config = ConfigFactory.load() // Automatically loads the application.conf
    val ktorConfig = HoconApplicationConfig(config)
    val port = ktorConfig.property("ktor.application.port").getString().toInt()
    val host = ktorConfig.property("ktor.application.host").getString()

    embeddedServer(Netty, port = port, host = host, module = Application::module)
        .start(wait = true)
}
```

Reply:
ok i added an application properties i think it makes sense